test_conformance/compiler/test_build_helpers.cpp

```diff
@@ -422,6 +443,118 @@ int test_get_program_info(cl_device_id deviceID, cl_context context, cl_command_
     return 0;
 }
 
+int test_get_program_info_kernel_names(cl_device_id deviceID,
+                                       cl_context context,
+                                       cl_command_queue queue, int num_elements)
+{
+    int error = CL_SUCCESS;
+    size_t total_kernels = 0;
+    size_t kernel_names_len = 0;
+
+    clProgramWrapper program = nullptr;
+
+    // 1) Program without build call
+    //    Query CL_PROGRAM_NUM_KERNELS and check that it fails with
+    //    CL_INVALID_PROGRAM_EXECUTABLE Query CL_PROGRAM_KERNEL_NAMES and check
```

Comment:
The comments seem to be missing punctuation, e.g.
```suggestion
    //    CL_INVALID_PROGRAM_EXECUTABLE.  Query CL_PROGRAM_KERNEL_NAMES and check
```

Reply:
Corrected